config/locales/en_LK.yml

```diff
@@ -52,28 +52,51 @@ en-LK:
   bs_below_200_copy:
     numerator: "Patients with RBS/PPBS <200, FBS<126, or Hba1c <7.0 at their last visit in the last 3 months"
     numerator_fbs: "Patients with FBS <126mg/dL, or HbA1c <7% at their last visit in the last 3 months"
+    numerator_dm: "Patients with RBS/PPBS <200, FBS<126 or Hba1c <7.0 at their last visit in the last 3 months"
+    numerator_dm_fbs: "Patients with FBS <126mg/dL or HbA1c <7% at their last visit in the last 3 months"
     reports_card_title: "Blood Sugar <200"
     reports_card_title_fbs: "Blood sugar controlled"
+    report_card_lower_bar: "Blood Sugar <200"
+    report_card_lower_bar_fbs:  "FBS <126mg/dL or HbA1c <7%"
+    denominator: "%{diagnosis} patients assigned to %{facility_name}, registered before the last 3 months. Dead and lost to follow-up patients are excluded."
```

Comment:
We can use "diabetes_denominator_copy" in en.yml which has the same text as needed here. The denominator would always remain same in the indicator calculations, so we don't need to change it.